### ISSUE TITLE
problem with setup

### ISSUE THREAD

**Issue by qxzusero** [UnamSanctam/UnamWebPanel#179]
it's just writing Endpoint is up and running. This page is shown since no data was posted during the request or the data posted was invalid.

**Comment by UnamSanctam** [comment]
Yes that's correct, then the miners should be able to connect to that page. That page is just for miners to connect to.

**Comment by qxzusero** [comment]
how can i connect miners? i should create new build?

> Yes that's correct, then the miners should be able to connect to that page. That page is just for miners to connect to.

**Comment by qxzusero** [comment]
it amazes me that you do such things for free, a great miner with a flexible setup, so also a panel that I learned about only today, so you are also responsible in a matter of minutes, I hope that you earn well

**Comment by qxzusero** [comment]
it's still remaining, i rebuild it, set to api, launch from many PCs with good gpu, though 2miners what that mining but panel still ignore it

**Comment by qxzusero** [comment]
> And port 80 (and port 443 if you use SSL/https) are open?

yea, at my server all ports are opened

**Comment by UnamSanctam** [comment]
Make sure that your `/db/unamwebpanel.db` file (and parent folder) are writable as well. Do you enter `http://SERVERIP/api/endpoint.php` into "API Endpoint URL"?

**Comment by qxzusero** [comment]
yea i did it for monero and etc

**Comment by UnamSanctam** [comment]
Are you able to go into "Configuration" and create a new configuration (enter `{}` as the data)? If you can find the configuration after creating it in the "Edit Configurations" then it's able to write to your database.
